### ISSUE TITLE
unknown error

### ISSUE THREAD

**Issue by fullbl** [wummel/linkchecker#646]
Hello, I've got this error when running on Debian 8!

```

INFO 2016-04-07 12:50:40,534 MainThread Checking intern URLs only; use --check-extern to check extern URLs.
 1 thread active,     0 links queued,    0 links in   0 URLs checked, runtime 1 seconds


********** Oops, I did it again. *************

You have found an internal error in LinkChecker. Please write a bug report
at https://github.com/wummel/linkchecker/issues
and include the following information:
- the URL or file you are testing
- the system information below

When using the commandline client:
- your commandline arguments and any custom configuration files.
- the output of a debug run with option "-Dall"

Not disclosing some of the information above due to privacy reasons is ok.
I will try to help you nonetheless, but you have to give me something
I can work with ;) .

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/linkcheck/director/checker.py", line 104, in check_url
    line: self.check_url_data(url_data)
    locals:
      self = <local> <Checker(CheckThread-https://www.***.com, started 140046890358528)>
      self.check_url_data = <local> <bound method Checker.check_url_data of <Checker(CheckThread-https://www.***.com, started 140046890358528)>>
      url_data = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
  File "/usr/lib/python2.7/dist-packages/linkcheck/director/checker.py", line 120, in check_url_data
    line: check_url(url_data, self.logger)
    locals:
      check_url = <global> <function check_url at 0x7f5f399f7b90>
      url_data = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      self = <local> <Checker(CheckThread-https://www.***.com, started 140046890358528)>
      self.logger = <local> <linkcheck.director.logger.Logger object at 0x7f5f36cf9a50>
  File "/usr/lib/python2.7/dist-packages/linkcheck/director/checker.py", line 52, in check_url
    line: url_data.check()
    locals:
      url_data = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      url_data.check = <local> <bound method HttpUrl.check of <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>>
  File "/usr/lib/python2.7/dist-packages/linkcheck/checker/urlbase.py", line 424, in check
    line: self.local_check()
    locals:
      self = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      self.local_check = <local> <bound method HttpUrl.local_check of <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>>
  File "/usr/lib/python2.7/dist-packages/linkcheck/checker/urlbase.py", line 442, in local_check
    line: self.check_connection()
    locals:
      self = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      self.check_connection = <local> <bound method HttpUrl.check_connection of <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>>
  File "/usr/lib/python2.7/dist-packages/linkcheck/checker/httpurl.py", line 135, in check_connection
    line: self.send_request(request)
    locals:
      self = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      self.send_request = <local> <bound method HttpUrl.send_request of <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>>
      request = <local> <PreparedRequest [GET]>
  File "/usr/lib/python2.7/dist-packages/linkcheck/checker/httpurl.py", line 165, in send_request
    line: self._send_request(request, **kwargs)
    locals:
      self = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      self._send_request = <local> <bound method HttpUrl._send_request of <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>>
      request = <local> <PreparedRequest [GET]>
      kwargs = <local> {'allow_redirects': False, 'timeout': 60, 'verify': True, 'stream': True}
  File "/usr/lib/python2.7/dist-packages/linkcheck/checker/httpurl.py", line 172, in _send_request
    line: self._add_ssl_info()
    locals:
      self = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      self._add_ssl_info = <local> <bound method HttpUrl._add_ssl_info of <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>>
  File "/usr/lib/python2.7/dist-packages/linkcheck/checker/httpurl.py", line 199, in _add_ssl_info
    line: self.ssl_cert = httputil.x509_to_dict(cert)
    locals:
      self = <local> <https link, base_url=u'https://www.***.com', parent_url=None, base_ref=None, recursion_level=0, url_connection=None, line=0, column=0, page=0, name=u'', anchor=u'', cache_url=https://www.***.com>
      self.ssl_cert = <local> None
      httputil = <global> <module 'linkcheck.httputil' from '/usr/lib/python2.7/dist-packages/linkcheck/httputil.pyc'>
      httputil.x509_to_dict = <global> <function x509_to_dict at 0x7f5f39d0fe60>
      cert = <local> <OpenSSL.crypto.X509 object at 0x7f5f35a9af10>
  File "/usr/lib/python2.7/dist-packages/linkcheck/httputil.py", line 35, in x509_to_dict
    line: from requests.packages.urllib3.contrib.pyopenssl import get_subj_alt_name
    locals:
      requests = <not found>
      requests.packages = <not found>
      requests.packages.urllib3 = <not found>
      requests.packages.urllib3.contrib = <not found>
      requests.packages.urllib3.contrib.pyopenssl = <not found>
      get_subj_alt_name = <not found>
ImportError: cannot import name get_subj_alt_name
System info:
LinkChecker 9.3
Released on: 16.7.2014
Python 2.7.9 (default, Mar  1 2015, 12:57:24) 
[GCC 4.9.2] on linux2
Requests: 2.4.3
Qt: 4.8.6 / PyQt: 4.11.2
Modules: QScintilla, Sqlite, Gconf
Local time: 2016-04-07 12:50:42+002
sys.argv: ['/usr/bin/linkchecker', 'https://www.***.com']
LANG = 'it_IT.utf8'
Default locale: ('it', 'UTF-8')

 ******** LinkChecker internal error, over and out ********
WARNING 2016-04-07 12:50:42,311 CheckThread-https://www.***.com internal error occurred

```

**Comment by anarcat** [comment]
this is fixed in #656

**Comment by dpalic** [comment]
Thank you for the issue report. Sadly this project is dead, and a new team is around with https://github.com/linkcheck/linkchecker 
for more details please see: #708
**Also please close this issue** and report it freshly on the new repo https://github.com/linkcheck/linkchecker/issues if your issue still persists
